src/components/AppProvider/AppProvider.tsx

```diff
@@ -36,6 +37,8 @@ export interface AppProviderProps extends AppBridgeOptions {
   linkComponent?: LinkLikeComponent;
   /** Custom logos and colors provided to select components */
   theme?: Theme;
+  /** For toggling settings */
+  settings?: Settings;
```

Comment:
Are we good with this prop name? Other options could be `features`, `flags`, etc.

Reply:
`flags` might be a better indicator of its intended use, but I'm comfortable with `settings`

Reply:
bikeshedding: flags or features feel better to me - "betas" while being our internal name for some of this stuff implies unfinished while settings feels a bit too permanent.